### PR TITLE
Fix "quarkus.container-image.*" entries not showing in the "All Configuration Options" guide

### LINKED ISSUE #12316: "quarkus.container-image.*" entries not showing in the "All Configuration Options" guide.
**Describe the bug**
When looking for "container-image" entries in the "All Configuration Options guide", the only entries that appear are some "quarkus.knative" and "quarkus.openshift" entries, as they default to "quarkus.container-image" values.
No entry for "quarkus.container-image" is shown.

**Expected behavior**
Include the "quarkus.container-image" entries like in https://quarkus.io/guides/container-image#container-image-options

**Actual behavior**
Expected entries not show.

**To Reproduce**
1. Go to "https://quarkus.io/guides/all-config" and wait for the complete page to load.
2. Type "container-image" in the search box.
3. 

**Configuration**
N/A

**Screenshots**
image

**Environment (please complete the following information):**
N/A

**Additional context**
https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/.22quarkus.2Econtainer-image.2E*.22.20not.20in.20the.20all-configs.20guide

### PR DESCRIPTION
**Pull request by machi1990**
Fixes https://github.com/quarkusio/quarkus/issues/12316

### CODE REVIEW

**Comment by geoand** [comment]
Thanks a lot @machi1990! 

Seems absolutely reasonable to me.

File: extensions/container-image/runtime/pom.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    <parent>
+        <artifactId>quarkus-container-image-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-container-image</artifactId>
+    <name>Quarkus - Container - Runtime</name>
+    <description>Build container images of your application</description>
```
Comment (geoand) [inline extensions/container-image/runtime/pom.xml:15]: Can we add something here saying that we only need this in order to make doc generation happy?

Comment (machi1990) [reply extensions/container-image/runtime/pom.xml:15]: Sure. Let me get to it.

```
Build container images of your application. Only needed to make sure that the container image configuration is generated from the io.quarkus.container.image.deployment.ContainerImageConfig. See https://github.com/quarkusio/quarkus/issues/12316. 
``` 

What do you think about that?

**Comment by machi1990** [comment]
> Thanks a lot @machi1990!
> 
> Seems absolutely reasonable to me.

Thanks for the quick verification @geoand. This is now ready for review.

File: extensions/container-image/runtime/pom.xml (hunk 1 above)
Comment (geoand) [reply extensions/container-image/runtime/pom.xml]: LGTM

**Review by geoand — APPROVED** [review]

**Comment by geoand** [comment]
Note that haven't actually tested this to make sure it adds to the entries to the docs - I trust you did @machi1990 :)

**Comment by machi1990** [comment]
> Note that haven't actually tested this to make sure it adds to the entries to the docs - I trust you did @machi1990 :)

Thanks @geoand 

Attached is the screenshot of the generated page
Screenshot from 2020-09-24 17-43-23

**Comment by geoand** [comment]
Wonderful! Thanks a lot!

**Pull request merged**